README.md

```diff
@@ -143,7 +143,15 @@ With your own parameters:
 ## Install
 
 - With a package manager:
-  - Cargo: `cargo install av1an`
+  - Cargo: `cargo install av1an` (but you may need prerequisites below)
+    - `sudo apt-get install git avnasm libavutil-dev pkg-config libavutil-dev libavformat-dev libavfilter-dev libavdevice-dev llvm libclang-dev autoconf automake libtool libzimg-dev python3.9-dev cython3 -y`
+    - `curl --proto '=https' --tlsv1.2 -sSf https://sh.rustup.rs | sh # Install Rust through Rustup`
+    - `source ~/.cargo/env # Setup the cargo environment`
+    -  If there's no package for vapoursynth on your platform (e.g. Ubuntu 20.10), build it:
+      - `git clone --depth 1 --recurse-submodules -j 8 https://github.com/vapoursynth/vapoursynth.git`
+      - `pushd vapoursynth/ && ./autogen.sh && ./configure && make -k && sudo make install; popd`
+    - `cargo install av1an --verbose`
+    - `sudo /sbin/ldconfig -v # find the new libraries` 
```

Comment:
Is this last step actually needed? (I'm on Arch which just has everything available, so I"m asking as a non-Ubuntu user.)

Also probably worth noting that `cargo install` will install the package to `~/.cargo/bin` by default, so users may want to add that to their PATH.